src/Modal.jsx

```diff
@@ -88,15 +88,29 @@ class Modal extends React.Component {
     return null;
   }
 
-  render = () => (
-    <div>
+  render = () => {
+    
+    const {
+      wrapperProps,
+      className,
+      visible,
+      onClickBackdrop,
+      onOK,
+      onCancel,
+      ...other
+    } = this.props;
+
+    return <div
```

Comment:
Stylistic note: please wrap this whole JSX block in parentheses to make eslint happy?